stock_packaging_usability_ul/__manifest__.py

```diff
@@ -1,37 +1,23 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Stock Packaging Usability UL module for Odoo
-#    Copyright (C) 2014 Akretion (http://www.akretion.com).
-#    @author Alexis de Lattre <alexis.delattre@akretion.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2014-2018 Akretion (http://www.akretion.com).
+# @author Alexis de Lattre <alexis.delattre@akretion.com>
```

Comment:
Remove @author

Reply:
Is it really a problem ?

Reply:
It looks weird and does not have real meaning (from my perspective): does it have a special signification for you?